src/components/atoms/ButtonNG/ButtonNG.scss

```diff
@@ -0,0 +1,121 @@
+@import "scss/constants.scss";
+
+.ButtonNG {
+  display: inline-block;
+
+  padding: 0.75em;
+  margin: $spacing--xs;
+  width: 100%;
+  max-width: 275px;
+
+  border-radius: $border-radius--xl;
+  border: none;
+  background-image: none;
+
+  font-style: normal;
+  font-size: 1rem;
+  font-weight: 500;
```

Comment:
```suggestion
  font-weight: $font-weight--500;
```

Reply:
Let's hope some day `$font-weight--500` doesn't become the value of `800`